src/llm/http_llm_calculator.cc

```diff
@@ -510,36 +509,15 @@ class HttpLLMCalculator : public CalculatorBase {
             // Unary scenario
             if (!this->request->isStream()) {
                 OVMS_PROFILE_SCOPE("Unary generation cycle");
-                std::vector<ov::genai::GenerationOutput> generationOutput = this->generationHandle->read_all();
-                RET_CHECK(generationOutput.size() >= 1);
-                std::sort(generationOutput.begin(), generationOutput.end(), [=](ov::genai::GenerationOutput& r1, ov::genai::GenerationOutput& r2) {
+                std::vector<ov::genai::GenerationOutput> generationOutputs = this->generationHandle->read_all();
+                RET_CHECK(generationOutputs.size() >= 1);
+                std::sort(generationOutputs.begin(), generationOutputs.end(), [=](ov::genai::GenerationOutput& r1, ov::genai::GenerationOutput& r2) {
```

Comment:
```suggestion
                std::sort(generationOutputs.begin(), generationOutputs.end(), [](ov::genai::GenerationOutput& r1, ov::genai::GenerationOutput& r2) {
```
You do not need to copy local variables to the lambda function since you do not use local variables.